.buildkite/scripts/bintray_publish.sh

```diff
@@ -26,16 +26,22 @@ echo "--- :habicat: Publishing all Habitat CLI artifacts in Bintray"
 version=$(buildkite-agent meta-data get "version")
 
 ########################################################################
-# Linux Publish
-release=$(buildkite-agent meta-data get "hab-release-linux")
-echo "--- :linux: Publishing Linux 'hab' ${version}-${release} on Bintray"
+# Linux x86-64-linux Publish
+release=$(buildkite-agent meta-data get "hab-release-x86_64-linux")
+echo "--- :linux: Publishing x86-64-linux 'hab' ${version}-${release} on Bintray"
 publish "https://api.bintray.com/content/habitat/${bintray_repository}/hab-x86_64-linux/${version}-${release}/publish"
 
+########################################################################
+# Linux x86-64-linux-kernel2 Publish
+release=$(buildkite-agent meta-data get "hab-release-x86_64-linux-kernel2")
+echo "--- :linux: :two: Publishing x86-64-linux-kernel2 'hab' ${version}-${release} on Bintray"
+publish "https://api.bintray.com/content/habitat/${bintray_repository}/hab-x86_64-linux-kernel2/${version}-${release}/publish"
```

Comment:
I think we're at the point where we want to refactor the `publish` function to take `repository`, `target`, `version`, and `release` arguments, and generate the complete URL internally. Otherwise, we've duplicated the URL creation logic four times.

Reply:
👍 Will refactor